Tests/ParameterEncoderTests.swift

```diff
@@ -226,7 +226,7 @@ final class URLEncodedFormEncoderTests: BaseTestCase {
         let result = Result<String, Error> { try encoder.encode(parameters) }
 
         // Then
-        XCTAssertEqual(result.value, "a=a")
+        XCTAssertEqual(result.success, "a=a")
```

Comment:
This is a bit of a surprising change to me. Has `error` also been refactored to `failure`? I also think I feel these are less intuitive than the previous. `value` and `error`. Could you speak to this change a bit?

Reply:
As we discussed, these changes just make the testing `Result` extensions match the internal ones. The internal extensions have been renamed both to match likely naming from possibly language features, as well as preventing users from seeing the "has internal access level" error on the old properties when updating code.